metadata-ingestion/src/datahub/ingestion/source/sql/postgres.py

```diff
@@ -138,18 +143,43 @@ def create(cls, config_dict, ctx):
         config = PostgresConfig.parse_obj(config_dict)
         return cls(config, ctx)
 
+    def get_inspectors(self) -> Iterable[Inspector]:
+        # This method can be overridden in the case that you want to dynamically
+        # run on multiple databases.
+        url = self.config.get_sql_alchemy_url()
+        logger.debug(f"sql_alchemy_url={url}")
+        engine = create_engine(url, **self.config.options)
+        with engine.connect() as conn:
+            if self.config.database and self.config.database != "":
+                inspector = inspect(conn)
+                yield inspector
+            else:
+                # pg_database catalog -  https://www.postgresql.org/docs/current/catalog-pg-database.html
+                # exclude template databases - https://www.postgresql.org/docs/current/manage-ag-templatedbs.html
+                databases = conn.execute(
+                    "SELECT datname from pg_database where datname not in ('template0', 'template1')"
+                )
+                for db in databases:
+                    if self.config.database_pattern.allowed(db["datname"]):
```

Comment:
what did database_pattern do previously for postgres?

Reply:
It didn't exist. Only `database` config was there